config/zones/CA-BC.yaml

```diff
@@ -15,22 +15,22 @@ emissionFactors:
   lifecycle:
     unknown:
       - datetime: '2019-01-01'
-        source: 'assumes 87% hydro, 5% biomass, 4% gas, 2% wind and 2% from various sources'
+        source: assumes 87% hydro, 5% biomass, 4% gas, 2% wind and 2% from various sources
         value: 56
   direct:
     unknown:
       - datetime: '2019-01-01'
-        source: 'assumes 87% hydro, 5% biomass, 4% gas, 2% wind and 2% from various sources'
+        source: assumes 87% hydro, 5% biomass, 4% gas, 2% wind and 2% from various sources
         value: 28.6
 isLowCarbon:
   unknown:
     - datetime: '2019-01-01'
-      source: 'assumes 87% hydro, 5% biomass, 4% gas, 2% wind and 2% from various sources'
+      source: assumes 87% hydro, 5% biomass, 4% gas, 2% wind and 2% from various sources
       value: 0.94
 isRenewable:
   unknown:
     - datetime: '2019-01-01'
-      source: 'assumes 87% hydro, 5% biomass, 4% gas, 2% wind and 2% from various sources'
```

Comment:
Are these changes okay? Is it due to the formatting? @VIKTORVAV99

Reply:
Perfectly fine, as long as it passes prettier it's good to go.